src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -214,11 +220,21 @@ private IRazorPage GetLayoutPage(ViewContext context, string layoutPath)
             var layoutPageResult = _viewEngine.FindPage(context, layoutPath);
             if (layoutPageResult.Page == null)
             {
+                _logger.LogError(
```

Comment:
When we spoke about this earlier today, I think the plan was to drop this - it's similar to `Html.Partial` \ templates etc which we aren't logging.